### PR TITLE
feat: extend runInBackground on application

### PR DESCRIPTION
**Pull request by fengmk2**
use mz-modules/sleep instead of ko-sleep

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by fengmk2** [comment]
在 app 下确实有这个需求 https://github.com/eggjs/examples/pull/18#discussion_r102673174

File: lib/application.js [hunk 1]
```diff
@@ -157,6 +156,16 @@ class Application extends EggApplication {
   }
 
   /**
+   * Run generator function in the background
+   * @see Context#runInBackground
+   * @param {Generator} scope - generator function, the first args is an anonymous ctx
+   */
+  runInBackground(scope) {
```
Comment (atian25) [inline lib/application.js]: 要增加对 aa 的支持不？

Comment (fengmk2) [inline lib/application.js]: scope 是 async function 默认是支持的。

我加个 test case on node 7

Comment (atian25) [inline lib/application.js]: 那 jsdoc 加下

Comment (atian25) [inline lib/application.js]: `@param {Generator} scope` 这个要改吧?

**Pull request merged**